conf/contributors.xml

```diff
@@ -1324,6 +1355,8 @@ Please keep the list in alphabetical order by last name!
  name="Ingolfur Edvardsson"
  trac="ingolfured"
  github="ingolfured"/>
+<contributor
+ name="Ingólfur Eðvarðsson"/>
 <contributor
```

Comment:
Same person with the previous one? May combine?

You may use ```altnames="Ingólfur Eðvarðsson"```.

Reply:
it's the same person, from Iceland. My former student

Reply:
Yeah, we may mannually combine it

These two entries also existed (at different places) in original list. The sorting brought them next to each other.